implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs

```diff
@@ -614,6 +615,7 @@ impl SessionReplacer for InletSessionReplacer {
 
             self.node_manager
                 .access_control(
+                    Some(GlobalResource::TcpInlet),
```

Comment:
Can we improve the clarity here? It's a little bit misleading to have Resource&GlobalResource. Maybe Resource=Address and GlobalResource=ResourceType? WDYT?